code/modules/clothing/spacesuits/spacesuits.dm

```diff
@@ -109,3 +109,50 @@
 		E.status &= ~ ORGAN_SPLINTED
 		user << "The suit stops supporting your [E.name]."
 	supporting_limbs = list()
+
+/obj/item/clothing/suit/space/explorer
```

Comment:
I am sure that most of those vars are already defined at /obj/item/clothing/suit/space and /obj/item/clothing/head/helmet/space. And already discussed on discord, the sprite does not really match our theme, was it was desinated for tg's lavaland. Besides, what is the purpose of this when there are the regular voidsuit and hardsuit, does mining really needs another space suit?